### ISSUE TITLE
The histogram and the result table don't match

### ISSUE THREAD

**Issue by DeepakSenapati** [AlexeyZatsepin/Android-ORM-benchmark#7]
The write and read columns are probably the other way around in results table. The results table shows the realm is best for read operations while the histograms show that its the case for write operations.

**Comment by AlexeyZatsepin** [comment]
Yeah, thank you for your feedback, histogram simple read and simple write was named vice versa.
I will check it
